spec/data_services_api/dataset_spec.rb

```diff
@@ -12,29 +12,33 @@
     VCR.eject_cassette
   end
 
-  it 'should have an id' do
-    _(@dataset.id).wont_be_nil
-  end
-
-  it 'should have a label' do
-    _(@dataset.label).must_match(/UK house price index/)
-  end
-
-  it 'should have a description' do
-    _(@dataset.description).must_match(/A Data Cube of UK house price/)
-  end
-
-  it 'should describe its own structure as a set of aspects' do
-    # VCR.use_cassette( name, :record => :new_episodes) do
-    structure = @dataset.structure
-    _(structure).wont_be_nil
-    _(structure).must_respond_to :size
-
-    structure.each do |aspect|
-      _(aspect.id).wont_be_nil
-    end
-    # end
-  end
+  # This feature is currently unavailable from SAPINT
```

Comment:
I would just delete these tests, and leave a comment to explain that to future readers